index.html

```diff
@@ -229,26 +229,15 @@ <h3>
             <dt id="data-integrity-1" class="spec">Verifiable Credential Data Integrity 1.0</dt>
             <dd>
               <p>
-This specification defines how to express proofs of integrity, such as digital
-signatures or proofs of existence, for bounded documents, such as verifiable
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This specification defines how to express proofs of integrity for verifiable
+credentials using a number of concrete serializations for each of the defined
+syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group.
               </p>
+              <p class="draft-status"><b>Draft state:</b> No Draft </p>
 
-              <p class="draft-status"><b>Draft state:</b>
+              <p class="input-documents"><b>Input documents:</b></p>
```

Comment:
I am a little concerned that there are multiple layers being mentioned without clarification of how they relate to expressing "proofs of integrity" - containers (JWTs, JWPs, etc), algorithms (BBS+, etc.), and concrete curves (secp256k1, ED25519, etc.)...

Reply:
Yeah, agree with @Sakurann ... we might want to be specific about the layers:

* Container Formats
  * Data Integrity 
  * VC-JWT
  * VC-JWP
* Cryptosuites (a fair number of these can be mapped to an input document)
  * JWS
  * EdDSA
  * NIST ECDSA
  * Koblitz ECDSA
  * NIST RSA
  * BBS+
  * PGP
  * GOST DSA
  * SM9 IBSA

Cryptographic algorithms and primitives are only to be taken from IETF and other standards organizations (e.g., SECG, X9, etc.)

Reply:
the word "Cryptosuites " is somewhat misleading, when "JWS" supports "EdDSA + NIST + Koblitz" ...

Reply:
- https://github.com/openpgpjs/openpgpjs#performance
- https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms

Reply:
JWS is a "kitchen sink" cryptosuite -- it includes everything (except for the things it doesn't... like GOST and SM9. :)

Reply:
Just want to call out that if we believe key formats should be tied to container formats and cryptosuites than I think we'll need to add those to the list. Although I'm personally in favor of not defining them here and relying on key conversion techniques so that an X.509 cert or a JWK set published as a resource on the web, or a DID Document with verification methods could all be used with the same cryptosuites or container formats and key formats remain an implementation considerations.